appveyor.yml

```diff
@@ -5,13 +5,9 @@ environment:
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
   matrix:
   - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-    Purpose: BuildingAndPackaging
+    Purpose: PesterCI
   - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-    Purpose: BuildingAndTesting
-
-# Stop all jobs in the matrix if any one them fail
-matrix:
-  fast_finish: true
+    Purpose: AdvancedPesterTests_xUnit_Packaging
```

Comment:
> AdvancedPesterTests

How about `ElevatedPesterTests`

Reply:
Could be but keep in mind that it is not only the admin tests, when the `feature` keyword is specified in a commit then it it also `'Slow', 'Feature', 'Scenario'` categories.

Reply:
Yes, it only indicates running Pester tests in an elevated process, not implying what tags of tests are run.

Reply:
Actually, `PesterCI` part only runs a sub set of PR CI tests -- it doesn't run tests with tag `-Tags
@('CI', 'RequireAdminOnWindows', 'RequireSudoOnUnix')`, or when `[Feature]` is sepecifed, it runs `'slow'`, `'feature'`, `'scenario'` tests as long as they don't have `'RequireAdminOnWindows'`. So to be precise, this purpose probably can be renamed to `UnelevatePesterTests`.